validator/validators/performance/performance_stats.go

```diff
@@ -61,3 +62,45 @@ func CalculateMetricStatisticsBasedOnDataAndPeriod(data []float64, dataPeriod fl
 		Period:  int(dataPeriod / float64(length)),
 	}
 }
+
+func CalculateMetricStatisticsWindows(datapoints []types.Datapoint, dataPeriod float64) Stats {
```

Comment:
This is more or less the same as the func above - which is what Id expect given there is no windows specific logic here.
Can we just create the `data[]` in the caller of this and invoke the func above? That way we dont need this here.

Reply:
Removed this function. Now we generate data[] for `types.Datapoint` variable then just use the `CalculateMetricStatisticsBasedOnDataAndPeriod` function